packages/clerk-js/src/core/clerk.ts

```diff
@@ -137,22 +139,31 @@ export default class Clerk implements ClerkInterface {
     this.#isReady = true;
   };
 
-  public signOutOne = async (
-    signOutCallback?: SignOutCallback,
-  ): Promise<void> => {
-    if (this.#environment?.authConfig.singleSessionMode) {
-      return this.signOut(signOutCallback);
+  public signOut: SignOut = async (
```

Comment:
🔧 Since we are now combining the methods for V3 let's unit test them as well.

Reply:
Thanks Sokrati, will cover everything with unit tests once we agree on the final api

Reply:
Cool, let's wrap this up tomorrow since the API is set now.

Reply:
Done!